builder

```diff
@@ -80,20 +80,36 @@ test "$(find "${kernel_source_dir}" -maxdepth 0 -type d -empty 2>/dev/null)" &&
 # configure kernel
 test -f "${kernel_source_dir}/.config" || sed -e "s/-rkt-v1/${kernel_version_suffix}/g" "${kernel_config}" >"${kernel_source_dir}/.config"
 
+function patch_kernel() {
+  filename=$1
```

Comment:
Nitpick: we try to follow https://google.github.io/styleguide/shell.xml, i.e. 

* use `local` variables in functions
* prefer `[[` over `[`

Also, `shellcheck` complains about a couple of things (e.g. missing `"` quotes; not all from your change).

I'll try to get back to issue with build images / assets later this week.